benchmarks/src/main/scala/zio/zmx/package.scala

```diff
@@ -0,0 +1,30 @@
+package zio
+
+package object zmx {
+  def spawn(i: Int): ZIO[Any, Nothing, Fiber.Runtime[Nothing, Unit]] =
```

Comment:
```suggestion
  def spawn(i: Int):UIO[Unit] =
```

Reply:
updated in https://github.com/zio/zio-zmx/pull/135/commits/cab61eb7f96d55c11127a95f3dc32f732ffa020f